GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/IPC/VFSForGitNotification.h

```diff
@@ -13,10 +15,12 @@ typedef NS_ENUM(NSInteger, Identifier)
 @interface VFSForGitNotification : NSObject
 
 @property (assign, readonly) Identifier identifier;
-@property (copy, readonly, nonnull) NSString *title;
-@property (copy, readonly, nonnull) NSString *message;
+@property (copy, readonly) NSString *title;
+@property (copy, readonly) NSString *message;
 
-+ (BOOL) tryCreateNotification:(VFSForGitNotification *_Nonnull *_Nonnull) notification
-                          from:(NSDictionary *_Nonnull) jsonMessage
-                         error:(NSString *_Nonnull *_Nonnull) error;
++ (BOOL)tryValidateMessage:(NSDictionary *_Nonnull)jsonMessage
```

Comment:
With NS_ASSUME_NONNULL_BEGIN, now you shouldn't need _Nonnull decorators in the method parameters

Reply:
Removed it for object references. As discussed - still need it for double pointers.